### ISSUE TITLE
False positive: property_enum_coverage

### ISSUE THREAD

**Issue by blandford** [bilelmoussaoui/gobject-lint#57]
This is possibly too non-standard, but maybe detectable.

I have a widget: XwdOverlayDock, and an XwdOverlayDockItem within it. They are defined in the same file. There are properties on both the main widget and the item. Goblint is warning about one of the props not being used — however it's being used by the other class

```
src/xwd-overlay-dock.c:387:1: warning: Property enum value 'DOCK_PROP_CHILD' is declared but never installed in xwd_overlay_dock_item_class_init [property_enum_coverage]
```

You can see these in https://gitlab.gnome.org/jrb/crosswords/-/blob/main/src/xwd-overlay-dock.c?ref_type=heads
(look for `DOCK_PROP_*` and `ITEM_PROP_*`)

**Comment by bilelmoussaoui** [comment]
I am not sure why the properties enum in that file didn't become named and used for casting prop_id, it feels like not all fixes were applied somehow as well

**Comment by bilelmoussaoui** [comment]
Ah you applied the rule before we made it fix more stuff. I would suggest reverting that commit and re-running the rule maybe?

**Comment by bilelmoussaoui** [comment]
Or maybe i can detect those cases and upgrade them as well, let me give it a try first
